### PR TITLE
Generate documentation from VAST CLI

### PR DESCRIPTION
**Pull request by dominiklohmann**

### CODE REVIEW

**Comment by dominiklohmann** [comment]
I've done a few changes to automate this better.

> Then there's one more thing: I think you can full generate the `_index.md` files in `/content/cli`, no?

Some have additional meta information at the top. I'll have to think of a way to integrate this properly.

**Comment by mavam** [comment]
> I've done a few changes to automate this better.

Looks good. Definitely a more maintainable solution than manually duplicating the command hierarchy.

> Some have additional meta information at the top. I'll have to think of a way to integrate this properly.

Right, my hunch is that the meta data can also be autogenerated.

File: scripts/generate-cli.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                dedent(
+                    f"""\
+                    ---
+                    title: {" ".join(command)}
+                    menuTitle: {command[-1] if len(command) > 1 else "CLI"}
+                    {"" if len(command) > 1 else 'pre: <b><i class="fas fa-terminal"></i> </b>'}
+                    ---
+
+                    {{{{< synopsis "{'/'.join(command)}" >}}}}
+                    {{{{% documentation "{'/'.join(command)}" %}}}}
```
Comment (mavam) [inline scripts/generate-cli.py]: Why does one shortcode start with ` {{{{<` and the other with `{{{{%`?

Comment (dominiklohmann) [reply scripts/generate-cli.py]: Because the first one renders HTML and the second one Markdown.

Comment (mavam) [reply scripts/generate-cli.py]: Ah!

**Comment by mavam** [comment]
Do you plan to add anything else to this PR? It looks mergeable to me. 👍

**Comment by dominiklohmann** [comment]
> Do you plan to add anything else to this PR?

I'm done.

**Pull request merged**